handbook/ce/license_keys.md

```diff
@@ -0,0 +1,46 @@
+# Creating and maintaining license keys for customers
```

Comment:
@dadlerj could we add a high level point here that CE should always work with Sales for the creation of all license keys for not yet signed accounts?